pkg/kubelet/util/ipv6_linux.go

```diff
@@ -0,0 +1,40 @@
+//go:build linux
+// +build linux
+
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	utilsysctl "k8s.io/component-helpers/node/util/sysctl"
+)
+
+const sysctlAllDisableIPv6 = "net/ipv6/conf/all/disable_ipv6"
+
+// IsIPv6Supported determines whether the host supports ipv6 by check the global ipv6 support sysctl.
+// We use it to determine whether the pod should support ipv6.
+// But there are going to be edge cases we don't get right,
+// e.g. the user may have IPv6 enabled in the host netns,
+// but for some reason disable IPv6 in the pod netns.
+// But because kubelet cannot actually know about the loopback interface of the pod,
+// we cannot do something completely correct,
+// so we can only do simple and basically correct things.
+func IsIPv6Supported() (bool, error) {
+	// TODO This should be injected by the caller.
```

Comment:
it is always questionable when the new code comes with TODO in it. Can you please clarify when this TODO is planned to be addressed? If not any time soon - maybe it is better to convert TODO into the explanaition of what may be wrong and how this can be improved in future

Reply:
Once this PR is merged, we can proceed to address this TODO. I will modify this comment later.

Reply:
Just to clarify, you are going to fix comment before you merge, correct? Only a few days till the code freeze so the follow up may not be in this release and this TODO may be there for a significant amount of time

Reply:
The TODO comments have been modified. Do you think the current comments are reasonable? This shouldn't cause any issues; as thockin mentioned, it's just a minor matter.